packages/@n8n/api-types/src/dto/credentials/credentials-get-many-request.dto.ts

```diff
@@ -0,0 +1,22 @@
+import { Z } from 'zod-class';
+
+import { booleanLiteral } from 'dto/common';
+
+export class CredentialsGetManyRequestQuery extends Z.class({
+	/**
+	 * Adds the `scopes` field to each credential which includes all scopes the
+	 * requesting user has in relation to the credential, e.g.
+	 *     ['credential:read', 'credential:update']
+	 */
+	includeScopes: booleanLiteral.optional(),
```

Comment:
maybe we should update the body/query parsing code to automatically make these boolean, so that we don't need to do string comparison in the controllers?